packages/react/src/button/Button.tsx

```diff
@@ -58,12 +90,24 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
         position="relative"
         rounded="sm"
         transition="colors"
-        {...recipe(props)}
+        {...recipe({
+          ...props,
+          colorScheme: finalColorScheme,
+          variant: finalVariant,
+        })}
+        data-disabled={props.disabled}
       >
         <Comp ref={composedRef}>
-          {leftSection}
-          <Slottable>{children}</Slottable>
-          {rightSection}
+          {isLoading ? (
```

Comment:
we can't do this otherwise `Slot` will break

let's address this in a separate PR completely and change this back to how it was before